modules/javafx.controls/src/test/java/test/javafx/scene/control/ListViewMouseInputTest.java

```diff
@@ -384,4 +385,15 @@ private boolean isAnchor(int index) {
         assertNotNull(sm.getSelectedItems().get(0));
         assertNotNull(sm.getSelectedItem());
     }
+
+    @Test public void testClickWithNullSelectionModelDoesNotThrowNPE() {
+        listView.setSelectionModel(null);
+
+        StageLoader loader = new StageLoader(listView);
+
+        assertDoesNotThrow(()  -> VirtualFlowTestUtils.clickOnRow(listView, 2));
+
+        loader.dispose();
+    }
```

Comment:
the stageloader is not disposed if the test fails - so a safer route is to keep it in a field which is then disposed in  @ after (which we decided to do years ago, but tend to forget occasionally :), see f.i. ListViewKeyInputTest

BTW: good to explicitely create the stageloader and __not__ rely on the one created implicitly created by the flowUtils!